### ISSUE TITLE
Move ckeditor5-specific webpack rules to the CKEditor5WebpackPlugin

### ISSUE THREAD

**Issue by ma2ciek** [ckeditor/ckeditor5-dev#58]
Thanks to the recently discovered webpack option we can move the ckeditor5-specific rules to the `ckeditor5-webpack-plugin`.

Code, that can add loader:
```
compiler.plugin( "normal-module-factory", ( nmf ) => {
	nmf.plugin( "after-resolve", ( options, done ) => {
		data.loaders.unshift( pathToLoader );
		done( null, options );
	} );
} );
```
I'm not sure if this will work with `compiler.plugin( 'after-resolvers', ... )` 

Old `webpack.config.js` part:
```js
module: {
	rules: [
		{
			// test: **/ckeditor5-*/theme/icons/*.svg
			test: /ckeditor5-[^/]+\/theme\/icons\/[^/]+\.svg$/,
			use: [ 'raw-loader' ]
		},
		{
			// test: **/ckeditor5-*/theme/**/*.scss
			test: /\.scss$/,
			use: [ 'style-loader', 'css-loader', 'sass-loader' ]
		}
	]
},
```

**Comment by ma2ciek** [comment]
AFAIK this issue is outdated. We used to declaring explicitly every loader in the `webpack.config.js` file. Am I right @Reinmar?

**Comment by Reinmar** [comment]
I still don't know. I'd keep this open and see where time gets us.

**Comment by Reinmar** [comment]
I mean – right now we put these rules in every config. Sometimes we use different rules if, e.g., we want to split CSS to a separate file. Our plugin shouldn't be that advanced to handle all those options because you can compile those files in many ways – there's CSS and SVG, minification, spriting, etc. Let's leave this to other developers. But at the same time, if we can provide some reasonable defaults in our plugin, it'd be nice to do so because it would simplify some devs work.

**Comment by ma2ciek** [comment]
I agree. We can provide the default loaders if no one for svg or scss is provided.

**Comment by Reinmar** [comment]
Can we? Let's remember that settings for loaders come with a path matcher. We can't easily analyse existing loaders and act only when some CKE5's stuff would not be handled by them. Unless... we somehow can. Anyway – I presume this will be a complicated topic so that's why I don't want to spend too much time on it :D Eventually, we may decide that it simply doesn't make sense to do anything as everyone handles this on their own.

**Comment by Reinmar** [comment]
I guess we're not going to do that at this stage. It would create a lot of confusion – why sometimes you need to configure stuff and at other occasions you don't.

However, I still like the idea very much, so I'll keep this ticket open.

Also, I started thinking – what if some envs like CRA2 allows extending webpack by loading some extra plugins, but don't allow other extensions? Then, this would be a great option to make building CKEditor 5 with those envs much much simpler (no ejection needed). So that can be a reason to get back to this ticket.

**Comment by ma2ciek** [comment]
Hm, adding a plugin to CRA2 that would do all the stuff seems quite nice, but OTOH this plugin would need to create a lot of magic around it and this would make extending, customizing and debugging potential problems much harder IMHO.
